### PR TITLE
display dm group name

### PR DESCRIPTION
**Pull request by MaxShv**

### CODE REVIEW

File: src/schema/resolvers.js [hunk 1]
```diff
@@ -131,15 +133,30 @@ module.exports = {
...
       return message;
     },
-    insertGroup: async (root, { workspace, members, name, oneToOne, projectId }, { user }) => {
+    insertGroup: async (root, { workspace, members, name, oneToOne, projectId },
+      { mongo: { Groups }, user }) => {
+      let group = {};
+
+      if (oneToOne) {
+        const dmQuery = { workspace, members: { $all: members }, oneToOne: true };
+        group = await Groups.findOne(dmQuery);
```
[the remark this answers is not in the document]
Comment (etyp) [reply src/schema/resolvers.js]: Please either add basic tests for this or make sure to manually test all cases.

File: src/utils/users.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+
+
+export const getDisplayNamesForUsers = (thisUsers, allWorkspaceUsers) => {
```
[the remark this answers is not in the document]
Comment (MaxShv) [reply src/utils/users.js]: line 16 outputs email address if user doesn't have first name yet.
It didn't work for you?

File: src/schema/resolvers.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (MaxShv) [reply src/schema/resolvers.js]: 👍

File: src/schema/resolvers.js [hunk 3]
```diff
@@ -335,12 +338,13 @@ module.exports = {
       { mongo: { Workspaces, Users }, user }) => {
       if (oneToOne) {
         // exclude myself (requesting userId)
-        const groupUsers = await Users.find({ $and: [{ _id: { $in: members } },
+        const memberWithoutMyself = members.filter(m => m !== user._id);
+        await Users.find({ $and: [{ _id: { $in: members } },
```
Comment (etyp) [inline src/schema/resolvers.js]: We're still doing 2 Users.find - can be limited to just one.

**Pull request merged**